dayOne/Numbers.java

```diff
@@ -0,0 +1,70 @@
+package dayOne;
+
+import java.util.Scanner;
+
+public class Numbers {
+	public static boolean checkPrime(int n) {
+		if(n<=1)return false;
+		if(n==2)return true;
+		for(int i=2;i<=n/2;i++)
```

Comment:
As mentioned in the PrimeNumbers program, try optimizing this logic even further